_data/monthly-calls.yml

```diff
@@ -1,5 +1,22 @@
 videos:
-  - title: Conducting user research with people with disabilities
+  - title: Designing a repeatable research process with people with disabilities
+    subtitle: Managing and sustaining our user research
+    description: |
+      We continued a discussion from the October monthly call about ongoing research on how
+      the design system works for users with disabilities. We presented more
+      [research findings](https://github.com/uswds/uswds/wiki/Usability-research-findings-for-Q3-2023)
+      and talked about what those results mean for USWDS components. We also defined
+      [best research practices](https://github.com/uswds/uswds/wiki/Usability-testing-day-checklist)
+      and plans to continue, improve, and share findings from this work.
+    date: November 2023
+    id: EQ8Nb60xgXA?si=eHcGR5WjpOVeDc07
+    event_link: https://digital.gov/event/2023/11/16/uswds-monthly-call-november-2023/
+    missing_script: true
```

Comment:
The new `missing_script` data field can be used when we have all items for the MC ready but the script hasn't been uploaded to digital.gov yet.